exercises/custom-set/custom_set_test.exs

```diff
@@ -8,121 +8,275 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule CustomSetTest do
   use ExUnit.Case
 
-  # Tip: to make your set work with the Set module be sure that your set value
-  # is a tuple with as first element `CustomSet`. A record with tag `CustomSet`
-  # will meet this requirement.
-
   # @tag :pending
-  test "delete" do
-    assert Set.equal?(Set.delete(CustomSet.new([3,2,1]), 2), CustomSet.new([1,3]))
-    assert Set.equal?(Set.delete(CustomSet.new([3,2,1]), 4), CustomSet.new([1,2,3]))
-    assert Set.equal?(Set.delete(CustomSet.new([3,2,1]), 2.0), CustomSet.new([1,2,3]))
-  end
+  describe "empty?" do
+    test "sets with no elements are empty" do
+      custom_set = CustomSet.new([])
+      assert CustomSet.empty?(custom_set) == true
+    end
 
-  @tag :pending
-  test "difference" do
-    assert Set.equal?(
-      Set.difference(CustomSet.new([1,2,3]), CustomSet.new([2,4])),
-      CustomSet.new([1,3]))
-    assert Set.equal?(
-      Set.difference(CustomSet.new([1,2.0,3]), CustomSet.new([2,4])),
-      CustomSet.new([1,2.0,3]))
+    test "sets with elements are not empty" do
+      custom_set = CustomSet.new([1])
+      assert CustomSet.empty?(custom_set) == false
+    end
   end
 
   @tag :pending
```

Comment:
I like the `describe` blocks. Are you at all worried that `@pending` these blocks will make too many test go red at one time? It seems ok since each is focused on one function.